soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3947,14 +3945,17 @@ void DrawRandoEditor(bool& open) {
                 ImGui::PushItemWidth(-FLT_MIN);
 
                 //Starting Age
-                //Disabled when Forest is set to Closed
-                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0);
-                const char* disableRandoStartingAgeText = "This option is disabled because \"Forest\" is set to \"Closed\".";
+                //Disabled when Forest is set to Closed or under very specific conditions
+                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0) || //If Forest is closed...
+                    (CVar_GetS32("gRandomizeDoorOfTime", 0) != 2) && //or Door of Time is Intended
+                    !(CVar_GetS32("gRandomizeShuffleOcarinas", 0)) && //and ocarinas are shuffled
+                    !(CVar_GetS32("gRandomizeLogicRules", 0)); // and logic is glitchless
```

Comment:
Is that comment supposed to say "and ocarinas *aren't* shuffled"? I'm having trouble understanding why Shuffled Ocarinas would be a problem here, seems like not shuffling them would be the more problematic case yeah?

Reply:
Yea, whoops. Will fix that.